Healthy/Classes/Modules/Login/LoginViewController.xib

```diff
@@ -124,33 +125,31 @@
+                                            <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
+                                            <buttonConfiguration key="configuration" style="plain"/>
                                             <connections>
-                                                <action selector="didTapSignInWithGoogle:" destination="-1" eventType="touchUpInside" id="asq-w7-g4Q"/>
+                                                <action selector="didTapSignInWithGoogle:" destination="-1" eventType="touchUpInside" id="gz9-n0-2J5"/>
                                             </connections>
-                                        </view>
-                                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Bm9-X6-e61" userLabel="Facebook Button">
+                                        </button>
+                                        <button opaque="NO" contentMode="scaleAspectFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Bm9-X6-e61" userLabel="Facebook Button">
                                             <rect key="frame" x="69" y="0.0" width="44" height="44"/>
                                             <constraints>
+                                                <constraint firstAttribute="width" secondItem="Bm9-X6-e61" secondAttribute="height" id="7w1-ZX-LZ0"/>
                                                 <constraint firstAttribute="height" constant="44" id="EEQ-UD-rts"/>
-                                                <constraint firstAttribute="width" secondItem="Bm9-X6-e61" secondAttribute="height" multiplier="1:1" id="aAe-WG-dSO"/>
                                             </constraints>
                                             <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
-                                            <state key="normal" image="die.face.5.fill" catalog="system"/>
+                                            <buttonConfiguration key="configuration" style="plain"/>
                                             <connections>
-                                                <action selector="didTapSignInWithFacebook" destination="-1" eventType="touchUpInside" id="C79-aL-XIL"/>
+                                                <action selector="didTapSignInWithFacebook:" destination="-1" eventType="touchUpInside" id="S48-V5-DAK"/>
                                             </connections>
                                         </button>
                                     </subviews>
+                                    <constraints>
```

Comment:
Using a horizontal stack view with equal width alignment and add this to a parent stack with center distribution would serve better. You will need to add only one constraint for a button and when it comes to add new Sign-in with Apple, we won't need to add and new constraint.